### PR TITLE
New option added in Spanish

### PR DESCRIPTION
**Pull request by mbarbancho**

### CODE REVIEW

**Comment by mbarbancho** [comment]
Please review this, @casey-collab

File: espanol/canudos.md [hunk 1]
```diff
@@ -31,3 +31,5 @@ el Consejero:
 [Una bruja entró en el salón](bruja/bruja.md)
 
 [Subió a la torre](torre/torre.md)
+
+[Ordenó buscar al Capitán Trueno](capitan/trueno.md)
```
Comment (casey-collab) [inline espanol/canudos.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
@casey-collab, please take another look

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**